CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 ## Unreleased
 
 - Editor
- - Restored #995 Improve element selected on `textDocument/hover` (previously reverted) with a fix that keeps it working for Calva even after a syntax error is introduced.
+  - Restored #995 Improve element selected on `textDocument/hover` (previously reverted) with a fix that keeps it working for Calva even after a syntax error is introduced.
+  - Add clojure.core/fn to resolve-macro-as options and sort options
```

Comment:
Could you link #1094 in the end of the sentence?

Reply:
Done!